tests/acceptance/features/apiSpacesShares/shareOperations.feature

```diff
@@ -12,75 +12,63 @@ Feature: sharing
     And using spaces DAV path
 
 
-  Scenario: correct webdav share-permissions for received file with edit and reshare permissions
+  Scenario: correct webdav share-permissions for received file with edit permissions
     Given user "Alice" has uploaded file with content "foo" to "/tmp.txt"
-    And user "Alice" has shared file "/tmp.txt" with user "Brian"
+    And user "Alice" has sent the following resource share invitation:
+      | resource        | tmp.txt     |
+      | space           | Personal    |
+      | sharee          | Brian       |
+      | shareType       | user        |
+      | permissionsRole | File Editor |
     When user "Brian" gets the following properties of file "/tmp.txt" inside space "Shares" using the WebDAV API
       | propertyName          |
       | ocs:share-permissions |
     Then the HTTP status code should be "207"
     And the single response should contain a property "ocs:share-permissions" with value "3"
 
 
-  Scenario: correct webdav share-permissions for received group shared file with edit and reshare permissions
+  Scenario: correct webdav share-permissions for received group shared file with edit permissions
     Given group "grp1" has been created
     And user "Brian" has been added to group "grp1"
     And user "Alice" has uploaded file with content "foo" to "/tmp.txt"
-    And user "Alice" has created a share with settings
-      | path        | /tmp.txt    |
-      | shareType   | group       |
-      | permissions | update,read |
-      | shareWith   | grp1        |
+    And user "Alice" has sent the following resource share invitation:
+      | resource        | tmp.txt     |
+      | space           | Personal    |
+      | sharee          | grp1        |
+      | shareType       | group       |
+      | permissionsRole | File Editor |
     When user "Brian" gets the following properties of file "/tmp.txt" inside space "Shares" using the WebDAV API
       | propertyName          |
       | ocs:share-permissions |
     Then the HTTP status code should be "207"
     And the single response should contain a property "ocs:share-permissions" with value "3"
 
 
-  Scenario: correct webdav share-permissions for received file with edit permissions but no reshare permissions
+  Scenario: correct webdav share-permissions for received file with read permissions
     Given user "Alice" has uploaded file with content "foo" to "/tmp.txt"
-    And user "Alice" has shared file "tmp.txt" with user "Brian"
-    When user "Alice" updates the last share using the sharing API with
-      | permissions | update,read |
-    Then the HTTP status code should be "200"
-    And as user "Brian" file "/tmp.txt" inside space "Shares" should contain a property "ocs:share-permissions" with value "3"
-
-
-  Scenario: correct webdav share-permissions for received group shared file with edit permissions but no reshare permissions
-    Given group "grp1" has been created
-    And user "Brian" has been added to group "grp1"
```

Comment:
We don't have to reshare permission, after removing the share permission the left scenario will be duplicate so removed the scenarios. (same applied for other places too)